templates/akeneo/files/.platform.app.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes an application.
+
+# The name of this app. Must be unique within a project.
+name: akeneo
+
+# The toolstack used to build the application.
+type: php:7.2
+
+# Enable non-default extensions required by Akeneo
+runtime:
+    extensions:
+        - apcu
+        - imagick
+
+# The relationships of the application with services or other applications.
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the environment variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: "mysqldb:mysql"
+    elasticsearch: "elasticsearch:elasticsearch"
+
+# The configuration of app when it is exposed to the web.
+web:
+    locations:
+        "/":
+            root: "web"
+            passthru: "/app.php"
+
+# The size of the persistent disk of the application (in MB).
+disk: 3072
+
+# Environment and PHP variables
+variables:
+    php:
+        "date.timezone": "Europe/Paris"
```

Comment:
Does Akeneo need this to be set?  That's odd.  Most apps have their own reasonable default.

If this is to be kept, we should probably put a comment on it that users should change it to their home timezone.

Reply:
removed, don't think that's necessary